paladin-core/src/config/mod.rs

```diff
@@ -40,6 +40,10 @@ pub struct Config {
     /// Provides the URI for the AMQP broker, if the AMQP runtime is selected.
     #[arg(long, help_heading = HELP_HEADING, env = "AMQP_URI", required_if_eq("runtime", "amqp"))]
     pub amqp_uri: Option<String>,
+
+    /// Provides the routing key for workers to listen on, if the AMQP runtime
```

Comment:
`if the runtime...`? It feels like something is missing

Reply:
fixed